### ISSUE TITLE
Uncategorized v1.0.0-beta questions

### ISSUE THREAD

**Issue by jadudm** [GSA-TTS/FAC#1549]

**Comment by jadudm** [comment]
Q1: Good catch! And, we're pretty sure this was an error on our part, which was frozen in as part of our ATO review. However, we think those fields should only be in `federal_awards`. 

Q2.1: Yes. 

Q2.2: We think this is correct. Meaning, the `award_reference` is not a globally unique identifier; in all cases, it will need to be used with the `report_id` in order to link between these tables. We believe, at the time of this writing, that `ELECAUDITSID` is, for all intents and purposes, the equivalent linking value, and we will carry it forward into that field.

**Comment by jadudm** [comment]
Q1: I have removed these two fields from `/general`, which seems more correct than it was. Leaving the ticket open for discussion.

Q2: `award_reference` is essentially a row identifier. The logic around it is actually rather complex, as a linking value between submission elements. `report_id` is enough to find everything related to a single submission; `award_reference` is necessary to link a given award's `finding`s with the associated `finding_text` and `cap_text` for each finding. 

In short, I'm not sure if this ID means the same thing to both of us. We can document this more clearly. This is a good question, and I want to make sure we're understanding it fully. Leaving this open for discussion.

**Comment by jadudm** [comment]
I think we've addressed this fully, and our tables reflect it. Closing. Tickets can be opened to reflect any future work that is needed.
